settings.gradle.kts

```diff
@@ -55,6 +55,7 @@ var addOns =
         "grpc",
         "highlighter",
         "imagelocationscanner",
+        "insights",
```

Comment:
Does this need to be included elsewhere for weeklies, or is that a change for later?